docs/source/en/tasks/video_classification.mdx

```diff
@@ -0,0 +1,482 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Video classification
+
+[[open-in-colab]]
+
+Video classification is the task of assigning a label or class to an entire video. Videos are expected to have only one class for each video. Video classification models take a video as input and return a prediction about which class the video belongs to. These models can be used to categorize what a video is all about. A real-world application of video classification is action / activity recognition which is useful for fitness applications. It is also helpful for vision-impaired individuals especially when they are commuting.
+
+This guide will show you how to:
+
+1. Fine-tune [VideoMAE](https://huggingface.co/docs/transformers/main/en/model_doc/videomae) on a subset of the [UCF101](https://www.crcv.ucf.edu/data/UCF101.php) dataset.
+2. Use your fine-tuned model for inference.
+
+<Tip>
+
+See the video classification [task page](https://huggingface.co/tasks/video-classification) for more information about its associated models, datasets, and metrics.
+
+</Tip>
+
+Before you begin, make sure you have all the necessary libraries installed:
+
+```bash
+pip install -q pytorchvideo transformers evaluate -q
+```
+
+You will use [PyTorch Video](https://pytorchvideo.org/) (dubbed as `pytorchvideo`) to process and prepare the videos.
```

Comment:
Nit - PyTorchVideo is one word afaik. Not a big deal though (You have this multiple times, so up to you if you think its worth updating)

Reply:
Updated.